### PR TITLE
Configure React, use React maps components

### PR DESCRIPTION
**Pull request by mjenczmyk**

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -1,13 +0,0 @@
...
-      "standard",
-      "promise"
-  ],
-  "globals": {
-      "document": true,
-      "window": true,
-      "$": true,
-      "jQuery": true
-  }
-};
```
Comment (mjenczmyk) [inline .eslintrc.js]: Replaced by `.eslintrc`.

File: .eslintrc [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    "no-mixed-operators": "off",
+    "no-multiple-empty-lines": ["error", { "max": 1, "maxBOF": 1, "maxEOF": 1 }]
+  },
+  "globals": {
+    "document": true,
+    "window": true,
+    "$": true,
+    "jQuery": true
+  }
+}
```
Comment (mjenczmyk) [inline .eslintrc]: That's exactly your configuration with the exception of tabs size (2 instead of 4).

File: src/js/components/Maps.jsx [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  />
+  <Marker position={ universityLocation }>
+    <Popup>
+      <div>
+        <b>Instytut Fizyki im. A. Chełkowskiego</b><br/>ul. Uniwersytecka 4<br/>40-007 Katowice<br/>
+        <a href="http://if.us.edu.pl/">http://if.us.edu.pl/</a>
+      </div>
+    </Popup>
+  </Marker>
+</Map>
```
Comment (mjenczmyk) [inline src/js/components/Maps.jsx]: Replacing plain leaflet maps.

File: package.json [hunk 4]
```diff
@@ -26,14 +26,20 @@
...
+    "babel-eslint": "^7.2.3",
+    "babel-loader": "^7.1.2",
+    "babel-preset-es2015": "^6.24.1",
+    "babel-preset-react": "^6.24.1",
+    "babel-preset-stage-0": "^6.24.1",
     "copy-webpack-plugin": "^4.0.1",
     "css-loader": "^0.28.7",
     "eslint": "^4.6.1",
-    "eslint-config-standard": "^10.2.1",
+    "eslint-config-airbnb-base": "^12.0.0",
```
Comment (mrdannael) [inline package.json]: and also eslint-config-airbnb

File: src/js/components/Maps.jsx [hunk 5]
```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import { Map, Marker, Popup, TileLayer } from 'react-leaflet'
+import 'leaflet-sleep'
+
+const hotelLocation = [50.3133248, 18.7666173]
+
+export const HotelMap = () => <Map center={ hotelLocation } zoom={ 14 } sleep={ true } sleepNote={ false }>
```
Comment (mrdannael) [inline src/js/components/Maps.jsx]: bleeding eyes

**Pull request merged**